### PR TITLE
Overhaul README.md

### PR DESCRIPTION
**Pull request by TheBunnyMan123**

### CODE REVIEW

**Comment by TheBunnyMan123** [comment]
Ready for review

File: README.md
Comment (PoolloverNathan) [inline README.md:10]: link is a 404; should this link to actions until the project is approved?

Comment (TheBunnyMan123) [reply README.md:10]: Good idea

Comment (PoolloverNathan) [reply README.md:10]: would a link like this work for now? (this can be reverted once modrinth is approved)
```suggestion
Download - [<img src="https://img.shields.io/github/actions/workflow/status/Figura-Goofballs/GoofyPlugin/build.yml?labelColor=1bd96a">](https://github.com/Figura-Goofballs/GoofyPlugin/actions/workflows/build.yml)
```

**Comment by TheBunnyMan123** [comment]
The modrinth link will work as soon as it is accepted though, I checked

**Comment by PoolloverNathan** [comment]
Still waiting for Modrinth approval...

File: README.md
Comment (TheBunnyMan123) [reply README.md:10]: yeah until modrinth link is public lets do this

**Comment by TheBunnyMan123** [comment]
Merge when the modrinth is accepted. The modrinth badge will automatically work and look good

**Review by PoolloverNathan — APPROVED** [review]
just going to merge this even though modrinth is broken

**Pull request merged**
